aspnetcore/toc.md

```diff
@@ -229,85 +207,111 @@
 ## [Troubleshoot](xref:test/troubleshoot)
 
-# [Data access with EF Core](xref:data/index)
-## [Get started with Razor Pages and EF Core using Visual Studio](xref:data/ef-rp/intro)
-## [Get started with ASP.NET Core and EF Core using Visual Studio](data/ef-mvc/index.md)
-## [ASP.NET Core with EF Core - new database](https://docs.microsoft.com/ef/core/get-started/aspnetcore/new-db)
-## [ASP.NET Core with EF Core - existing database](https://docs.microsoft.com/ef/core/get-started/aspnetcore/existing-db)
-## [Get started with ASP.NET Core and Entity Framework 6](data/entity-framework-6.md)
-## [Azure Storage](data/azure-storage/index.md)
-### [Add Azure Storage by using Visual Studio Connected Services](https://azure.microsoft.com/documentation/articles/vs-azure-tools-connected-services-storage/)
-### [Get started with Blob storage and Visual Studio Connected Services](https://azure.microsoft.com/documentation/articles/vs-storage-aspnet5-getting-started-blobs/)
-### [Get Started with Queue Storage and Visual Studio Connected Services](https://azure.microsoft.com/documentation/articles/vs-storage-aspnet5-getting-started-queues/)
-### [Get Started with Table Storage and Visual Studio Connected Services](https://azure.microsoft.com/documentation/articles/vs-storage-aspnet5-getting-started-tables/)
+# Data access
+## Tutorials
+### EF Core with Razor Pages
+#### [Overview](xref:data/ef-rp/index)
+#### [Get started](xref:data/ef-rp/intro)
+#### [Create, Read, Update, and Delete](xref:data/ef-rp/crud)
+#### [Sort, filter, page, and group](xref:data/ef-rp/sort-filter-page)
+#### [Migrations](xref:data/ef-rp/migrations)
+#### [Create a complex data model](xref:data/ef-rp/complex-data-model)
+#### [Read related data](xref:data/ef-rp/read-related-data)
+#### [Update related data](xref:data/ef-rp/update-related-data)
+#### [Handle concurrency conflicts](xref:data/ef-rp/concurrency)
+### [EF Core with MVC, new DB](/ef/core/get-started/aspnetcore/new-db)
+### [EF Core with MVC, existing DB](/ef/core/get-started/aspnetcore/existing-db)
+### EF Core with MVC, long tutorial
+#### [Overview](xref:data/ef-mvc/index)
+#### [Get started](xref:data/ef-mvc/intro)
+#### [Create, Read, Update, and Delete](xref:data/ef-mvc/crud)
+#### [Sort, filter, page, and group](xref:data/ef-mvc/sort-filter-page)
+#### [Migrations](xref:data/ef-mvc/migrations)
+#### [Create a complex data model](xref:data/ef-mvc/complex-data-model)
+#### [Read related data](xref:data/ef-mvc/read-related-data)
+#### [Update related data](xref:data/ef-mvc/update-related-data)
+#### [Handle concurrency conflicts](xref:data/ef-mvc/concurrency)
+#### [Inheritance](xref:data/ef-mvc/inheritance)
+#### [Advanced topics](xref:data/ef-mvc/advanced)
+## [EF 6 with ASP.NET Core](xref:data/entity-framework-6)
+## Azure Storage with Visual Studio
+### [Connected Services](/azure/vs-azure-tools-connected-services-storage)
+### [Blob storage](/azure/vs-storage-aspnet5-getting-started-blobs/)
```

Comment:
storage is lowercase here, but it's uppercase in the following 2 links (I suspect it should be uppercase for all of them)